docs/user-guide/advanced-configuration/deploying-a-specific-version-of-awx.md

```diff
@@ -2,15 +2,17 @@
 
 There are a few variables that are customizable for awx the image management.
 
-| Name                | Description               | Default                                    |
-| ------------------- | ------------------------- | -----------------------------------------  |
-| image               | Path of the image to pull | quay.io/ansible/awx                        |
-| image_version       | Image version to pull     | value of DEFAULT_AWX_VERSION or latest     |
-| image_pull_policy   | The pull policy to adopt  | IfNotPresent                               |
-| image_pull_secrets  | The pull secrets to use   | None                                       |
-| ee_images           | A list of EEs to register | quay.io/ansible/awx-ee:DEFAULT_AWX_VERSION |
-| redis_image         | Path of the image to pull | docker.io/redis                            |
-| redis_image_version | Image version to pull     | latest                                     |
+| Name                         | Description               | Default                                    |
+| ---------------------------- | ------------------------- | ------------------------------------------ |
+| image                        | Path of the image to pull | quay.io/ansible/awx                        |
+| image_version                | Image version to pull     | value of DEFAULT_AWX_VERSION or latest     |
+| image_pull_policy            | The pull policy to adopt  | IfNotPresent                               |
+| image_pull_secrets           | The pull secrets to use   | None                                       |
+| ee_images                    | A list of EEs to register | quay.io/ansible/awx-ee:DEFAULT_AWX_VERSION |
+| redis_image                  | Path of the image to pull | docker.io/redis                            |
+| redis_image_version          | Image version to pull     | latest                                     |
+| init_container_image         | Path of the image to pull | quay.io/ansible/awx-ee                     |
+| init_container_image_version | Image version to pull     | value of DEFAULT_AWX_VERSION or latest     |
```

Comment:
We should consider adding `control_plane_ee_image` and `init_projects_container_image` to this table. Could you please add these parameters, or may I create another PR?

```suggestion
| Name                          | Description               | Default                                    |
| ----------------------------- | ------------------------- | ------------------------------------------ |
| image                         | Path of the image to pull | quay.io/ansible/awx                        |
| image_version                 | Image version to pull     | value of DEFAULT_AWX_VERSION or latest     |
| image_pull_policy             | The pull policy to adopt  | IfNotPresent                               |
| image_pull_secrets            | The pull secrets to use   | None                                       |
| ee_images                     | A list of EEs to register | quay.io/ansible/awx-ee:DEFAULT_AWX_VERSION |
| redis_image                   | Path of the image to pull | docker.io/redis                            |
| redis_image_version           | Image version to pull     | latest                                     |
| control_plane_ee_image        | Image version to pull     | quay.io/ansible/awx-ee:DEFAULT_AWX_VERSION |
| init_container_image          | Path of the image to pull | quay.io/ansible/awx-ee                     |
| init_container_image_version  | Image version to pull     | value of DEFAULT_AWX_VERSION or latest     |
| init_projects_container_image | Image version to pull     | quay.io/centos/centos:stream9              |
```